### PR TITLE
Add CITATION.cff

### LINKED ISSUE #17: Add CITATION.cff
CITATION.cff should be for the code only as the data is not stored in this repo.  So, this is a separate issue from #5.

### PR DESCRIPTION
**Pull request by Aariq**
Adds a CITATION.cff file that I created with [CFFINIT](https://citation-file-format.github.io/cff-initializer-javascript/#/).  This will help GitHub create a "cite this" button on our repository and will populate metadata for when the **code** is archived in Zenodo.  Note: this citation is only for the *code* and not the data, since the data isn't stored in this repo.

### CODE REVIEW

**Comment by UArizonaCampusLivingLabforGSI** [comment]
I reviewed this issue and think this makes sense for citing the code. I did notice that there is a link in the code to a news story by the Udall Center. I am wondering if having the campus living lab website would be a more appropiate link to include.

**Comment by Aariq** [comment]
I can't find the Campus Living Lab homepage.  The Udall Center article is what comes up if I google "University of Arizona Campus Living Lab".

**Comment by UArizonaCampusLivingLabforGSI** [comment]
The website is gicampuslivinglab.arizona.edu<http://gicampuslivinglab.arizona.edu/> and is set to launch early January. This will be where the link for the shinyapp is and we can add the final link anytime after the launch.

If we need a link to describe the project, it might be best to use the PIF report highlight but I am not sure what the primary use is.
________________________________
From: Eric R. Scott ***@***.***>
Sent: Thursday, December 14, 2023 2:28 PM
To: UArizonaGSICampusLivingLab/gsi-wrangling-workflow ***@***.***>
Cc: Buzzard, Vanessa - (vbuzzard) ***@***.***>; Review requested ***@***.***>
Subject: [EXT]Re: [UArizonaGSICampusLivingLab/gsi-wrangling-workflow] Add CITATION.cff (PR #18)


External Email

I can't find the Campus Living Lab homepage. The Udall Center article is what comes up if I google "University of Arizona Campus Living Lab".

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: >-
+  Campus Living Lab Green Stormwater Infrastructure Data
+  Workflow
+message: >-
+  To cite the code in this repository, use the metadata in this
+  file. If you wish to use the data and cite it, please use 
+  citation information on the data request form 
+  <https://forms.gle/63qWCybhvHaHunuH6>.
+type: software
+version: 0.0.1
```
Comment (Aariq) [inline CITATION.cff]:
```suggestion
version: 0.1.0
```

**Pull request merged**